hydra-filters/src/test/java/com/addthis/hydra/data/filter/bundle/TestBundleFilterTimeRange.java

```diff
@@ -13,38 +13,69 @@
  */
 package com.addthis.hydra.data.filter.bundle;
 
+import com.addthis.bundle.core.Bundle;
 import com.addthis.bundle.util.AutoField;
+import com.addthis.codec.config.Configs;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
 import org.joda.time.format.DateTimeFormatter;
 import org.junit.Before;
 import org.junit.Test;
 
+import java.io.IOException;
 import java.text.DecimalFormat;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
+import static org.junit.Assert.*;
 import static org.mockito.Mockito.mock;
 
 public class TestBundleFilterTimeRange {
     AutoField timeIn ;
     DateTime utc;
     DecimalFormat df;
+    Bundle bundle;
+
+    static final String before1 = "time:TIME, before:2016-09-23, timeFormat:YYYY-MM-dd";
+    static final String before2 = "time:TIME, before:2015-09-25, timeFormat:YYYY-MM-dd";
+    static final String after1  = "time:TIME,  after:2016-09-21, timeFormat:YYYY-MM-dd";
+    static final String after2  = "time:TIME,  after:2017-09-21, timeFormat:YYYY-MM-dd";
```

Comment:
All constants above should be `private` too. Also the naming convention for constants are all CAPS. The names can be better too. For example, instead of `before1`, consider `BEFORE_160923` - it makes the test methods below easier to read.